CONTRIBUTING.md

```diff
@@ -150,6 +150,12 @@ $ ./configure && make -j8 test
 Make sure the linter is happy and that all tests pass. Please, do not submit
 patches that fail either check.
 
+You can check if the linter is happy by running:
```

Comment:
@diosney Yep, I’ve seen the thread that prompted this PR, and I’m in favour of having the information of how to run only the linter noted here – I’d just want to save folks the trouble of thinking they have to run the linter separately under normal circumstances.

I don’t feel too strongly about it, and if you want to keep it this way, I won’t be in the way of that. :)